src/components/page/WithGATracker/index.tsx

```diff
@@ -30,32 +43,47 @@ export const getGAusername = (): string => username;
  * @param {string} page the url string of the page view being tracked
  * @param {FlexObject} options tracking options for the page view
  */
-export const trackPage = (page: string, options: FlexObject = {}): void => {
-  GoogleAnalytics.set({
-    page,
-    ...options,
-  });
-  GoogleAnalytics.pageview(page);
+export const trackPage = (page: string, options?: FlexObject): void => {
```

Comment:
- I still think that `options` does not need to be of type `FlexObject`.
- I feel that it is weird that `options` is ... optional for `trackPage`.  Why does it need to be this way?  What would it mean to call `trackPage` and supply no options i.e. what would one be trying to achieve when they do that?

Reply:
As it is, if we didn't pass anything in as `options` then it would use the global value for `GA_CODE`, but we can make it required and only care about what is passed in with `options`.

Reply:
I think this is similar to `getGaCode` i.e. I think its better to have just one entry point for values to get into a function and that should be the params that the function accepts.